policykit/starterkits/dictator.txt

```diff
@@ -0,0 +1,124 @@
+{
+  "name": "Dictator",
+  "description": "One user is given complete access and control over all policies and actions.",
+  "platform_policies": [
+    {
+      "name": "All Platform Actions Pass",
+      "description": "All platform actions pass automatically",
+      "filter": "return True",
+      "initialize": "pass",
+      "check": "return PASSED",
+      "notify": "pass",
+      "success": "pass",
+      "fail": "pass"
+    }
+  ],
+  "constitution_policies": [
+    {
+      "name": "Dictator: Only Dictator's Constitution Actions Pass",
+      "description": "Only actions proposed by dictator pass",
+      "filter": "return True",
+      "initialize": "pass",
+      "check": "return FAILED",
+      "notify": "pass",
+      "success": "action.execute()",
+      "fail": "pass"
+    }
+  ],
+  "roles": [
+    {
+      "name": "Base User",
+      "description": "",
+      "is_base_role": true,
+      "user_group": "all",
+      "permission_sets": [
+        "view",
+        "propose"
+      ],
+      "permissions": [
+        "Can view boolean vote",
+        "Can view number vote",
+        "Can view platformactionbundle",
+        "Can view platformpolicybundle",
+        "Can view constitutionactionbundle",
+        "Can view constitutionpolicybundle",
+        "Can view policykit add role",
+        "Can view policykit delete role",
+        "Can view policykit edit role",
+        "Can view policykit add user role",
+        "Can view policykit remove user role",
+        "Can view policykit change platform policy",
+        "Can view policykit change constitution policy",
+        "Can view policykit remove platform policy",
+        "Can view policykit remove constitution policy",
+        "Can view policykit recover platform policy",
+        "Can view policykit recover constitution policy",
+        "Can view policykit add platform policy",
+        "Can view policykit add constitution policy",
+        "Can view policykit add community doc",
+        "Can view policykit change community doc",
+        "Can view policykit delete community doc",
+        "Can view policykit recover community doc"
+      ]
+    },
+    {
+      "name": "Dictator",
+      "description": "Has ability to propose and execute actions",
+      "is_base_role": false,
+      "user_group": "creator",
+      "permission_sets": [
+        "view",
+        "propose",
+        "execute"
+      ],
+      "permissions": [
+        "Can add boolean vote",
+        "Can change boolean vote",
+        "Can delete boolean vote",
+        "Can view boolean vote",
+        "Can add number vote",
+        "Can change number vote",
+        "Can delete number vote",
+        "Can view number vote",
+        "Can add platformactionbundle",
+        "Can add platformpolicybundle",
+        "Can add constitutionactionbundle",
+        "Can add constitutionpolicybundle",
```

Comment:
These will need to be updated to remove nonexistent permissions in the same way that https://github.com/amyxzhang/policykit/pull/448 did